mappings/net/minecraft/test/TestImplementation.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_6302 net/minecraft/test/TestImplementation
+	METHOD method_35932 timeout ()I
```

Comment:
In `TestFunction` this is `tickLimit`, I would try and keep these names matching.